engine.go

```diff
@@ -224,14 +232,55 @@ func (engine *Engine) GetDocumentType() string {
 	}
 }
 
+func (engine *Engine) channelControl() {
+	if engine.UseDefaultChannelControl {
```

Comment:
Como vai funcionar o `channelControl` quando o `UseDefaultChannelControl` for `false`? A declaração e o uso dele vai ser feito do lado da engine no próprio `EntryPoint`?

Reply:
Isso mesmo! Se `UseDefaultChannelControl` for definido como `false`, cabe ao dev implementar sua própria estrutura de controle de canais.